core/src/main/java/nl/nn/adapterframework/util/XmlUtils.java

```diff
@@ -1416,13 +1396,27 @@ public static String source2String(Source source, boolean removeNamespaces) thro
 		return writer.toString();
 	}
 
-	public static String removeNamespaces(String input) {
+	public static String removeNamespaces(String input) throws XmlException {
+		try (Reader reader = new StringReader(input)) {
+			XmlWriter xmlWriter = new XmlWriter();
+			ContentHandler handler = new NamespaceRemovingFilter(xmlWriter);
+			InputSource inputSource = new InputSource(reader);
+			parseXml(inputSource, handler);
+			return xmlWriter.toString();
+		} catch (Exception e) {
+			throw new XmlException(e);
+		}
+	}
+
+	public static Message removeNamespaces(Message input) throws XmlException {
 		try {
-			TransformerPool tp = getRemoveNamespacesTransformerPool(true,false);
-			return tp.transform(input,null);
+			XmlWriter xmlWriter = new XmlWriter();
+			ContentHandler handler = new NamespaceRemovingFilter(xmlWriter);
+			input.preserve();
```

Comment:
Is deze preserve nodig?

Reply:
Als je na een fout toch iets meet de oorspronkelijke input wil doen? Ik kan het hier weghalen, misschien beter op z'n plek inderdaad bij de aanroeper.

Reply:
Het was ook een over-het-hoofd-gezien overblijfsel van toen deze code nog de input retourneerde bij fouten. Wat we niet meer doen. :-)